packages/specification/electronic-prescription-service-api.yaml

```diff
@@ -446,7 +456,7 @@ paths:
                   description: |
                     An error response.
                   value:
-                    $ref: examples/spec/errors/example-a-validation-error-missing-field/Response-FhirError.json
+                    - $ref: examples/spec/errors/example-a-validation-error-missing-field/Response-FhirError.json
```

Comment:
I suspect the hyphen was added by mistake.

Reply:
Correct. I will remove.